.github/.licenserc.yaml

```diff
@@ -8,6 +8,7 @@ header:
     - '.gitignore'
     - '.gitattributes'
     - '.golangci.yml'
+    - '.golangci_br.yml'
```

Comment:
Shall we depend on the same lint rule together with tidb. We should avoid do more special log for br if not necessary

Reply:
At present, supported linters are different between tidb and br. disable-all is set by tidb, and then many linter are turned on. br set enable-all, and then turn off many linter.
We need to turn on check-static for br, and then adjust individual linter with tidb one by one later